eng/pipelines/templates/jobs/archetype-sdk-client-samples.yml

```diff
@@ -14,18 +14,22 @@ stages:
             pool.name: azsdk-pool-mms-ubuntu-2004-general
             image.name: MMSUbuntu20.04
             go.version: '1.17'
+            generate.bom: false
```

Comment:
Given this is the default value is it worth copying everywhere?

Reply:
You have a good point. I'll try it.